pom.xml

```diff
@@ -91,6 +91,11 @@
             <version>1.2.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.docker-java</groupId>
+            <artifactId>docker-java</artifactId>
+            <version>3.2.13</version>
+        </dependency>
```

Comment:
Why do we need this? Please add details in the PR description

Reply:
We want to add the ability to assemble a tarantool from a bundle and cache an image. Since ryuk removes images after testing (not configurable as it's a bug testcontainers/testcontainers-java#4702) - this results in long runs when retesting. And we need dockerclient for managing images directly.